### PR TITLE
Test: add linting support via Travis-CI

### PR DESCRIPTION
**Pull request by mbologna**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    - TMPDIR=~/tmp
+
+before_install:
+  - sudo apt install -y alien
+  - wget http://download.opensuse.org/repositories/home:/SilvioMoioli:/tools/SLE_12_SP1/x86_64/terraform.rpm
+  - sudo alien -i terraform.rpm
+  - wget http://download.opensuse.org/repositories/home:/SilvioMoioli:/tools/SLE_12_SP1/x86_64/terraform-provider-libvirt.rpm
+  - sudo alien -i terraform-provider-libvirt.rpm
+
+install: /bin/true
```
Comment (moio) [inline .travis.yml]: Dumb question: why is installation handled in `before_install` and not here?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+language: go
+
+env:
+  global:
+    - TMPDIR=~/tmp
+
+before_install:
+  - sudo apt install -y alien
+  - wget http://download.opensuse.org/repositories/home:/SilvioMoioli:/tools/SLE_12_SP1/x86_64/terraform.rpm
+  - sudo alien -i terraform.rpm
```
Comment (moio) [inline .travis.yml]: Is there any chance to run this in a SUSE container?

Comment (mbologna) [inline .travis.yml]: Travis supports only Ubuntu and macOS.

File: .travis.yml (hunk 1 above)
Comment (mbologna) [inline .travis.yml]: "In a before_install step, you can install additional dependencies required by your project such as Ubuntu packages or custom services."

https://docs.travis-ci.com/user/customizing-the-build/#The-Build-Lifecycle

File: .travis.yml (hunk 2 above)
Comment (MalloZup) [inline .travis.yml]: yop

**Comment by MalloZup** [comment]
nice pr :rocket:

**Comment by moio** [comment]
Thanks!

**Pull request merged**